pkg/kubelet/rkt/rkt.go

```diff
@@ -2449,3 +2449,18 @@ func convertKubePortMappings(portMappings []kubecontainer.PortMapping) ([]appcty
 
 	return containerPorts, hostPorts
 }
+
+// makeCapabilities creates string slices from Capability slices
+func makeCapabilities(capAdd []v1.Capability, capDrop []v1.Capability) ([]string, []string) {
```

Comment:
I moved it to `pkg/kubelet/container` to be shared by both dockertools and rkt. This doesn't seem like a function that non-kubelet packages will be interested in, and we will be deprecating dockertools and rkt in the future. Are you ok with that?